### DISCUSSION TITLE
Find the Column name with Spreadsheet

### DISCUSSION THREAD

**Discussion by sebviatour-iteamlu** [cfsimplicity/lucee-spreadsheet#353] [Q&A]
Hello,

Does it exist a function that return the name of the column (ex : "AA" or "C") and is it possible to store it into a variable ?

Thanks for your feedback.

Best Regards,

Sébastien

**Comment by cfsimplicity** [comment]
Currently, you can do it *unofficially* using the cell helper:
```
spreadsheet = New spreadsheet.Spreadsheet();
workbook = spreadsheet.new();
cell = {
	columnNumber: 27
	,rowNumber: 1
};
spreadsheet.setCellValue( workbook, "test", cell.rowNumber, cell.columnNumber );
cell.object = spreadsheet.getCellHelper().getCellAt( workbook, cell.rowNumber, cell.columnNumber );
cell.address = cell.object.getAddress().formatAsString();
cell.columnName = cell.address.ReReplace( "\d+", "" );
WriteDump( cell.columnName ); // AA
```
But really you shouldn't rely on the helpers directly, or any internal methods that aren't part of the public API.

**Reply by cfsimplicity** [reply]
Have created a ticket for an official method: https://github.com/cfsimplicity/spreadsheet-cfml/issues/354
